### ISSUE TITLE
Feature Request:  Add the same Heat "glow" effect as the native Thermostat card

### ISSUE THREAD

**Issue by mkanet** [KartoffelToby/better-thermostat-ui-card#158]
There is only one thing that the native Thermostat card does better than the "better-thermostat-ui-card".  The Heat "glow" effect (shown in the screenshot below) when Heating is turned on looks much nicer.  I would just use the native card instead of "better-thermostat-ui-card"; however, it's HVAC buttons for some reason are in a weird order that I could never get used to.

![image](https://github.com/KartoffelToby/better-thermostat-ui-card/assets/811008/fbe8598e-73ac-4e18-91c1-fe8ab3c92df4)

If it's not too much trouble, could you please add this visual effect?

**Comment by Drwatson10** [comment]
Yes, that visuel effect were very nice for me

**Comment by RYUcze** [comment]
+1

**Comment by mkanet** [comment]
@KartoffelToby are you still maintaining this repo?  Any chance you could add this great feature?

**Comment by KartoffelToby** [comment]
@mkanet  yes i still maintain it. But currently i Focus on the Integration. When 1.5 is released the ui will get some love from me.

Dont worry the glow effect will come :)

**Comment by mkanet** [comment]
Hi @KartoffelToby sorry to bug you again.  The integration was updated to version 1.5.1 a couple of weeks ago.  Any ideas when we will see this cosmetic change to this card?
